ui-components/textarea-with-emoji.md

```diff
@@ -0,0 +1,182 @@
+# Textarea with Emoji picker Component
+
+1. [General information](#general-info)
+2. [Setup](#setup)
+3. [Usage](#usage)
+
+## General information
+
+An emoji picker component is a user interface element that allows users to select emojis from a list or grid of available options. It's a popular feature in modern applications, messaging platforms, and social media websites, as emojis have become an essential part of online communication, adding emotion and expression to messages.
+
+The emoji picker typically appears as a small window or popup that contains a collection of emojis categorized into various groups like smileys and people, animals and nature, food and drink, etc. Users can scroll through the list or use search functionality to find the desired emoji and click or tap on it to insert it into the text input field, comment section, or wherever they need it.
+
+It could look like this component below:
+![emoji picker component](https://github.com/uptechteam/prism-athlete-fe/assets/13544983/5be21177-a78a-4901-b96d-d116dfb9305b)
```

Comment:
broken link

Reply:
yes, you don't have access here. Added a fix